XamarinCommunityToolkit/Behaviors/Animations/AnimationHelper.shared.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Xamarin.Forms;
+
+namespace XamarinCommunityToolkit.Behaviors.Animations
+{
+    public enum EasingType
```

Comment:
@jfversluis
jfversluis 3 days ago Member
Talked about this with @Char0394 its either doing this, or you have to specify the easing like this in your XAML: <behaviors:FadeAnimation Easing="{x:Static Easing.Linear}" Duration="100"/> because Easing is a class with static properties. Both is not great.

I think for the latter there might be some magic we can do for XAML, but let's keep it like this for now and revisit later